### PR TITLE
Release/v7.0.0

### PR DESCRIPTION
**Pull request by Manuthor**
---

### Added

- instantiated types in `statics.rs`
- `Serializable` trait

### Changed

- use `Shake256` to derive keys
- use constant generics instead of `GenericArray`
- make `EncryptedHeader` generic
- make `ClearTextHeader` generic
- use EAKEM (cf [MR](https://github.com/Cosmian/cover_crypt/pull/39))
- `UserSecretKey::x` is now a `HashSet` (`Partition`s are removed)
- `CoverCrypt::encaps()` now takes an `AccessPolicy`
- replace `Hc128` by `ChaCha12Rng` as RNG

### Removed

- `Metadata`

---

### CODE REVIEW

**Comment by ThibaudDauce** [comment]
Could we also merge https://github.com/Cosmian/cover_crypt/pull/55 in this release?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
 
 All notable changes to this project will be documented in this file.
 
+---
+
+## [7.0.0] - 2022-10-17
```
Comment (tbrezot) [inline CHANGELOG.md:7]: Maybe update the date

**Review by tbrezot — APPROVED** [review]

**Comment by Manuthor** [comment]
> Could we also merge #55 in this release?

done

File: CHANGELOG.md (hunk 1 above)
Comment (Manuthor) [reply CHANGELOG.md]: done

**Pull request merged**